include/bio/record.hpp

```diff
@@ -76,6 +76,18 @@ enum class field : uint64_t
     filter,    //!< FILTER field in Var I/O.
     info,      //!< INFO field in Var I/O.
     genotypes, //!< GENOTYPES in Var I/O.
+
+    // Fields unique to annotation io
+    chromStart = pos,
+    chromEnd,
+    score,
+    strand,
+    thickStart,
+    thickEnd,    //!< Ending position
+    itemRgb,     //!< An RGB value to determine the color of the displayed track in the browser.
+    blockCount,  //!< The number of blocks (exons) in the BED file.
+    blockSizes,  //!< A list of the block sizes, corresponding to blockCount.
+    blockStarts, //!< A list of block starts, relative to offset.
```

Comment:
I know these use `camelCase` in the specification, but it looks very strange to have that mixed with the other formatting styles in this library.
Can we change this to having `snake_case` or do you think that will confuse users?

Reply:
I could change it! I just did it this way because I wasn't sure if I should be consistent with the specs or with our code haha.

Reply:
Yeah, I think we should stick with the code style for now. I would love to have a table like this in the documentation at some point:

|  bio::field::                      |   bio::fasta  | bio::fastq | bio::vcf      | bio::bcf    | bio::sam  | bio::bed |
|---------------------------------|----------------|--------------|---------------|--------------|--------------|------------|
|  ::id == ::qname              |  description line   |   description line  |  ID  |  ID | QNAME | name   |
|  ::seq                              |  sequence data   |  sequence data  |  –     |  –   | SEQ      |   –  |
|  ::chrom == ::rname        |  –               |    --           |  CHROM  |  CHROM |  RNAME | chrom   |
| ::qual == ::mapq             |  –               | quality data | QUAL    | QUAL      | MAPQ    | – |
| ::pos == ::chrom_start    |  –               | –             | POS          | POS      |  POS   | chromStart |

...

So, instead of having individual documentation for all the fields, one big table with the format-specific terminology would be more helpful I think.